AttendanceSystem.Domain.Services/Tools/MockDataGenerator.cs

```diff
@@ -87,76 +60,97 @@ public MockDataGenerator(UserService userService, CourseService courseService, A
         "Education", "Law", "Health Sciences", "Arts", "Architecture", "Data Science", "Chemistry"
     };
 
-        public async Task GenerateMockData()
+        public async Task GenerateRealData()
```

Comment:
This is a great method! We may want to make this method idempotent in the future, but seeing as we don't have a lot of time, let's do that another time :)